server/manager/healthcheck/healthcheck.go

```diff
@@ -11,23 +12,26 @@ import (
 
 type HealthCheckManager struct {
 	db *gorm.DB
+	ctx context.Context
+	cancelFunc context.CancelFunc
 }
 
 func New(db *gorm.DB) *HealthCheckManager {
 	db.AutoMigrate()
-	healthCheckManager := &HealthCheckManager{db: db}
+	ctx, cancelFunc := context.WithCancel(context.Background())
+	healthCheckManager := &HealthCheckManager{db: db, ctx: ctx, cancelFunc: cancelFunc }
 	pubsub.UpdateConfigEvent.Sub(healthCheckManager.onUpdateConfig)
 	return healthCheckManager
 }
 
 func (m *HealthCheckManager) onUpdateConfig(updateConfig pubsub.UpdateConfig) {
 	config := updateConfig.Config
+	m.cancelFunc()
```

Comment:
↓の部分をこっちに持ってくるのがいい感じです
```suggestion
	m.cancelFunc()
	ctx, cancelFunc := context.WithCancel(context.Background())
	m.ctx = ctx
	m.cancelFunc = cancelFunc
```